.github/workflows/rust.yml

```diff
@@ -38,12 +38,26 @@ jobs:
           toolchain: "1.86.0"
           components: clippy,rustfmt
       - uses: Swatinem/rust-cache@9d47c6ad4b02e050fd481d890b2ea34778fd09d6 # v2
+      # https://github.com/actions/runner-images/issues/5459#issuecomment-1532856844
+      - name: Set up GNU patch on Windows
```

Comment:
Do we need this?

Reply:
I would like to have that for consistent behavior between target operating systems.